pkg/component/kubestatemetrics/monitoring.go

```diff
@@ -122,6 +123,7 @@ var (
 		monitoringMetricKubePodInfo,
 		monitoringMetricKubePodLabels,
 		monitoringMetricKubePodOwner,
+		monitoringMetricKubePodCreated,
```

Comment:
allowedMetrics list doesn't need to be adjusted. Our monex seed config scrapes everything from kube-state-metrics that is returned from it.